src/components/ColourModeSwitch.tsx

```diff
@@ -0,0 +1,21 @@
+import { Accessor, Setter, Show } from "solid-js";
+
+export default function ColourModeSwitch({
+  getter,
+  setter,
+}: {
+  getter: Accessor<boolean>;
+  setter: Setter<boolean>;
+}) {
+  // FIXME: This is temporary, and the styling, etc. should be improved.
+  return (
+    <button
+      onClick={() => setter(a => !a)}
+      class="absolute rounded-full bg-black p-2 transition-all dark:bg-white"
+    >
+      <Show when={getter()} fallback={<img src="/icons/sun.svg" class="invert" />}>
```

Comment:
Is it possible to set the value of `fallback` to it's own variable that's passed in rather than nesting components in this jarring manner?